data/json/items/armor/ppe.json

```diff
@@ -0,0 +1,439 @@
+    "environmental_protection": 4,
+    "delete": { "flags": [ "WATERPROOF", "RAINPROOF", "GAS_PROOF", "RAD_PROOF", "ELECTRIC_IMMUNE" ] },
+    "use_action": {
+      "type": "transform",
+      "menu_text": "button-down",
+      "msg": "You flip down the chemical hood, unzip what's left of the coat, and pull up your sleeves.",
+      "target": "mopp_jacket_undone_breached"
+    }
+  },
+  {
+    "id": "mopp_jacket_undone_breached",
+    "type": "ARMOR",
+    "copy-from": "mopp_jacket_undone",
+    "name": { "str": "breached JSLIST overcoat (unzipped)", "str_pl": "breached JSLIST overcoats (unzipped)" },
+    "description": "A waist-length overcoat with an integral chemical hood, an expandable pocket sewn into the left sleeve, and Velcro fastenings on the wrists and zipper, this ripped and torn jacket used to form half of the US military’s CBRN JSLIST overgarment.  Constructed with an inner layer of carbon discs to keep harmful chemicals from penetrating through the suit and a selectively permeable membrane meant to let perspiration escape, you would be ill advised to wear this particular garment in a biohazardous environment.  Even had its zipper not been undone, the hood lowered, and its sleeves rolled up, the entire thing is peppered with lacerations, allowing unimpeded access to any gas, vapor, or liquid contaminants you may encounter.",
+    "environmental_protection": 4,
+    "use_action": {
+      "type": "transform",
+      "menu_text": "zip-up",
+      "msg": "You pull the hood over your head, let down the cufs, and do your best to zip up what’s left of the coat.",
+      "target": "mopp_jacket_breached"
+    }
+  },
+  {
+    "id": "mopp_jacket_expended",
+    "type": "ARMOR",
+    "copy-from": "mopp_jacket",
+    "name": { "str": "expended JSLIST overcoat" },
+    "description": "A waist-length overcoat with an integral chemical hood, an expandable pocket sewn into the left sleeve, and Velcro fastenings on the wrists and zipper, this jacket forms half of the US military’s CBRN JSLIST overgarment. Constructed with an inner layer of carbon discs that keep harmful chemicals from penetrating through the suit and a selectively permeable membrane that lets perspiration escape, the garment’s meant to be either worn over a regular uniform or alone for maximum breathability.  This particular coat has been unsealed for quite a while, and its carbon layer and water-repellent coating have been exhausted; it will no longer block incoming vapor, gas, or liquid contaminants.  That being said, it can still protect you from any alpha or beta radiation particles you may encounter.",
+    "environmental_protection": 4,
+    "delete": { "flags": [ "WATERPROOF", "RAINPROOF", "GAS_PROOF" ] },
```

Comment:
Is the ``damage_verb`` inherited on this, so you can't keep the radproof element by opening an overcoat & letting it expire?